databricks-kube/src/crds/databricks_job.rs

```diff
@@ -45,7 +45,7 @@ impl Default for DatabricksJobStatus {
     fn default() -> Self {
         Self {
             latest_run_state: Some(RunState {
-                life_cycle_state: Some(RunLifeCycleState::NoRuns),
+                life_cycle_state: Some(RunLifeCycleState::Pending),
```

Comment:
Our customization has been removed from this enum. The default job state is now pending.